generators/exercises/sublist.py

```diff
@@ -0,0 +1,40 @@
+FUNC_PROTO = """\
+#include "vendor/unity.h"
+
+#include <stddef.h>
+#include <stdint.h>
+
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof((x)[0]))
+
+typedef enum {
+    UNEQUAL,
```

Comment:
Re my previous comment:
```suggestion
    UNEQUAL = 0,
```